This patch is needed to build successfully on PPC, where the linker is
evidently pickier about functions vs variables than on other Fedora
platforms.  Note it is already committed upstream, so we won't need it
in postgres 8.3.2 and later.


Index: configure.in
===================================================================
RCS file: /cvsroot/pgsql/configure.in,v
retrieving revision 1.546.2.3
diff -c -r1.546.2.3 configure.in
*** configure.in	14 Mar 2008 03:23:22 -0000	1.546.2.3
--- configure.in	28 Apr 2008 22:41:54 -0000
***************
*** 795,801 ****
  fi
  
  if test "$with_libxslt" = yes ; then
!   AC_CHECK_LIB(xslt, xsltLibxmlVersion, [], [AC_MSG_ERROR([library 'xslt' is required for XSLT support])])
  fi
  
  # for contrib/uuid-ossp
--- 795,801 ----
  fi
  
  if test "$with_libxslt" = yes ; then
!   AC_CHECK_LIB(xslt, xsltCleanupGlobals, [], [AC_MSG_ERROR([library 'xslt' is required for XSLT support])])
  fi
  
  # for contrib/uuid-ossp
